Shopify/tophat
# Remove `ios-deploy`

### What does this change accomplish?

This deletes the `ios-deploy` submodule and all dependencies on `ios-deploy`, which was used to support physical iOS devices running iOS 16 or earlier.  With two major iOS versions now supporting CoreDevice _via_ `devicectl`, there's no longer much need to keep support for older versions as testing on older versions declines.

### How have you achieved it?

By deleting the submodule and all references to `ios-deploy` in code.

### How can the change be tested?

Launch Tophat and ensure that you can still launch builds on devices running iOS 17 and newer.
